tests/test.json

```diff
@@ -0,0 +1,29 @@
+{
+  "metadata": {
+    "mapFileFormatVersion": 1,
+    "commandBasedBackendType":"SERIAL",
+    "defaultRecoveryRegister":"/IDN",
+    "delim":"\r\n"
+  },
+  "registers": {
+      "/cwFrequency":{"writeCmd":"SOUR:FREQ:CW {{x.0}}", "writeResp":"", "readCmd":"SOUR:FREQ:CW?", "readResp":"([0-9]+)\r\n"},
```

Comment:
Empty/unused parameters should be optional. We should leave them out to keep he map files as compact as possible. They have to be hand-crafted anyway.